packages/wonder-blocks-dropdown/components/select-box.js

```diff
@@ -127,9 +107,11 @@ export default class SelectBox extends React.Component<SelectBoxProps> {
                                 {children}
                             </LabelMedium>
                             <View style={[styles.spacing]} />
```

Comment:
Hmm - just realizing that we can use the new Strut component for stuff like this! Might be worth it for us to do a pass back through at some point (don't need to do it now).